libs/guid/CMakeLists.txt

```diff
@@ -4,7 +4,9 @@
 liblet(guid
   DEPENDS
     Mso::compilerAdapters
+    Mso::crash
     Mso::debugAssertApi
+    Mso::memoryApi
```

Comment:
The new dependencies create a circular dependency. The liblet compilation supposed to provide an error about it.

Should we put the new headers in the new guidUtils liblet?